### PR TITLE
ENT-4823: Add support for setting abortclasses and abortbundleclasses from augments (3.12.x)

### PR DESCRIPTION
**Pull request by nickanderson**

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thank you for submitting a PR! Maybe @craigcomstock can review this?

File: MPF.md [hunk 1]
```diff
@@ -974,6 +974,40 @@ For example:
...
+
+```
+{
+  "vars":{
+    "control_agent_abortclasses": [ "error.*", "abort.**" ]
+  }
+}
+```
+
+**History**: Added in 3.15.0, 3.12.3
```
Comment (craigcomstock) [inline MPF.md]: why is 3.15.0 included here?

Comment (nickanderson) [reply MPF.md]: It's the next version that this will be part of (after 3.14)

Comment (nickanderson) [reply MPF.md]: Oh you mean in this branch of docs? I haven't ever made sure that old branches do not mention newer ones. I can yank it out though if you think i should.

**Pull request merged**